scripts/gen-edpm-baremetal-kustomize.sh

```diff
@@ -0,0 +1,201 @@
+        neutron_public_interface_name: eth0
+        ctlplane_mtu: ${INTERFACE_MTU}
+        ctlplane_subnet_cidr: 24
+        ctlplane_gateway_ip: 192.168.122.1
+        ctlplane_host_routes:
+        - ip_netmask: 0.0.0.0/0
+          next_hop: 192.168.122.1
+        role_networks:
+        - InternalApi
+        - Storage
+        - Tenant
+        networks_lower:
+          External: external
+          InternalApi: internal_api
+          Storage: storage
+          Tenant: tenant
+        # edpm_nodes_validation
+        edpm_nodes_validation_validate_controllers_icmp: false
+        edpm_nodes_validation_validate_gateway_icmp: false
+
+        edpm_ovn_metadata_agent_DEFAULT_transport_url: ${EDPM_OVN_METADATA_AGENT_TRANSPORT_URL}
+        edpm_ovn_metadata_agent_metadata_agent_ovn_ovn_sb_connection: ${EDPM_OVN_METADATA_AGENT_SB_CONNECTION}
+        edpm_ovn_metadata_agent_metadata_agent_DEFAULT_nova_metadata_host: ${EDPM_OVN_METADATA_AGENT_NOVA_METADATA_HOST}
+        edpm_ovn_metadata_agent_metadata_agent_DEFAULT_metadata_proxy_shared_secret: ${EDPM_OVN_METADATA_AGENT_PROXY_SHARED_SECRET}
+        edpm_ovn_metadata_agent_DEFAULT_bind_host: ${EDPM_OVN_METADATA_AGENT_BIND_HOST}
+        edpm_chrony_ntp_servers:
+        - ${EDPM_CHRONY_NTP_SERVER}
+
+        dns_search_domains: []
+        edpm_ovn_dbs:
+        - ${EDPM_OVN_DBS}
+        registry_name: quay.io
+        registry_namespace: podified-antelope-centos9
+        image_tag: current-podified
+        edpm_ovn_controller_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-ovn-controller:{{ image_tag }}"
+        edpm_iscsid_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-iscsid:{{ image_tag }}"
+        edpm_logrotate_crond_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-cron:{{ image_tag }}"
+        edpm_nova_compute_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-compute:{{ image_tag }}"
+        edpm_nova_libvirt_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-libvirt:{{ image_tag }}"
+        edpm_ovn_metadata_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-neutron-metadata-agent-ovn:{{ image_tag }}"
+
+        gather_facts: false
+        enable_debug: false
+        # edpm firewall, change the allowed CIDR if needed
+        edpm_sshd_configure_firewall: true
+        edpm_sshd_allowed_ranges: ${EDPM_SSHD_ALLOWED_RANGES}
+        # SELinux module
+        edpm_selinux_mode: enforcing
+        plan: overcloud
+    - op: replace
+      path: /spec/roles/edpm-compute/networkAttachments
+      value: ${EDPM_NADS}
+    - op: replace
+      path: /spec/roles/edpm-compute/nodeTemplate/ansibleSSHPrivateKeySecret
+      value: ${EDPM_ANSIBLE_SECRET}
+EOF
+if [ "$EDPM_SINGLE_NODE" == "true" ]; then
+cat <<EOF >>kustomization.yaml
+    - op: remove
+      path: /spec/nodes/edpm-compute-1
+EOF
+elif [ "$EDPM_TOTAL_NODES" -gt 2 ]; then
```

Comment:
Should this be `-gt 1`?  Otherwise it seems you need at least 3 for `EDPM_TOTAL_NODES` to have multiple nodes created.

Reply:
I think there is a `DATAPLANE_SINGLE_NODE` var which has to be changed to `false` from the default `true`. I just copied it from what they've for pre-provisoned, though it could have been managed with one var. We delete one node L172-173 from the sample if  `DATAPLANE_SINGLE_NODE` is true